README.md

```diff
@@ -81,8 +81,8 @@ Total library size can depend on feature usage. Rough approximations and average
 
 |**Features** | **Flash (text,rodata,data)** | **RAM1,RAM2(dss,data)** |
```

Comment:
can we call out compiler for which these number are?

Reply:
I added compile conditions but not the compiler itself.